apps/apps.c

```diff
@@ -235,6 +235,19 @@ int ctx_set_verify_locations(SSL_CTX *ctx, const char *CAfile,
     return SSL_CTX_load_verify_locations(ctx, CAfile, CApath);
 }
 
+int ctx_set_ctlog_list_file(SSL_CTX *ctx, const char *path)
+{
+    if (path == NULL) {
+        if (SSL_CTX_set_default_ctlog_list_file(ctx) <= 0) {
+            BIO_puts(bio_err, "Failed to load default Certificate Transparency "
```

Comment:
Unfortunately, the default file path is in crypto/include/internal/cryptlib.h. Am I ok reaching into a crypto internal header from apps? The comparable code for loading the default root certificates doesn't print an error message on failure, which is also an option.